src/dr/evomodelxml/operators/NodeHeightOperatorParser.java

```diff
@@ -46,8 +46,8 @@ public class NodeHeightOperatorParser extends AbstractXMLObjectParser {
     public enum OperatorType {
         UNIFORM("uniform"),
         RANDOMWALK("random walk"),
-        SCALEROOT("scale root"),
-        SCALEALL("scale all internal");
+        SCALEROOT("scaleRoot"),
```

Comment:
Why the switch in this label to camel case?

Reply:
I was running into a parsing issue with this line below because I was shooting for the enum name in the xml "scale root" and not the value which doesn't have the space. It was just an xml error on my end from not reading the code all the way through. I  will revert the names back from camel case.
```
        OperatorType operatorType = OperatorType.UNIFORM;
        if (xo.hasAttribute(OPERATOR_TYPE)) {
            try {
                operatorType = OperatorType.valueOf(xo.getStringAttribute(OPERATOR_TYPE).trim().toUpperCase());
            } catch (IllegalArgumentException iae) {
                throw new XMLParseException("Unrecognised operator type attribute: " + xo.getStringAttribute(OPERATOR_TYPE));
            }
        }
```